CREDITS.md

```diff
@@ -195,17 +212,21 @@ Contributors to the Firecracker repository:
 - Sam Jackson <sam@clique.app>
 - Samuel Knox <knoxsa@oregonstate.edu>
 - Samuel Ortiz <sameo@linux.intel.com>
+- ScmTble <ScmTble@qq.com>
 - Sean Lavine <freewil@users.noreply.github.com>
 - Sebastien Boeuf <sebastien.boeuf@intel.com>
 - Serban Iorga <seriorga@amazon.com>
+- ShadowCurse <yegorlz@amazon.co.uk>
 - shakram02 <ahmedhamdyau@gmail.com>
 - Shen Jiale <shenjiale@baidu.com>
 - Shion Yamashita <shioyama1118@gmail.com>
 - singwm <singwm@amazon.com>
+- sladynnunes <snunes@usc.edu>
 - Sripracha <ramsri@amazon.com>
 - Stefan Nita <32079871+stefannita01@users.noreply.github.com>
 - StemCll <lydjotj6f@mozmail.com>
 - Sudan Landge <sudanl@amazon.com>
+- sudanl0 <119602619+sudanl0@users.noreply.github.com>
```

Comment:
Here too.

Reply:
For this one we would need an additional entry for Sudan:

```
Sudan Landge <sudanl@amazon.com> <119602619+sudanl0@users.noreply.github.com>
```

Reply:
fixed